src/component-library/CollateralScore/CollateralScore.tsx

```diff
@@ -0,0 +1,120 @@
+import { useMeter } from '@react-aria/meter';
+import { HTMLAttributes, ReactNode } from 'react';
+
+import { Severity } from '../utils/prop-types';
```

Comment:
NIT: Just a little question on this type - is it supposed to be specific to vault collateralization or generally any status? Because if the latter I would probably name it `status` which seems to me to match the possible values(error,warning,success) of this type better :)